src/com/tt/TrayUI.java

```diff
@@ -93,7 +93,7 @@ public void mouseClicked(MouseEvent e) {
                         JLabel pageStartLabel = new JLabel("<html><center><span style='color:white;font-size:18px;'>" + geoAPIKeyValue[0] + "</span><br/><hr/><span style='color:white;font-size:10px;'><i>" + LocalDate.now().getDayOfWeek() + " &nbsp;|&nbsp; WEEK OF THE MONTH: " + Calendar.getInstance().get(Calendar.WEEK_OF_MONTH) + "</i></span></center></html>", SwingConstants.CENTER);
 
                         mainPanel.add(pageStartLabel, BorderLayout.PAGE_START);
-                        JLabel pageEndLabel = new JLabel("<html><center><span style='color:white;font-size:15px;'><i>Feels like " + weatherAPIKeyValue[1] + "°C</i></span><p style='color:white;font-size:12px;'>High " + weatherAPIKeyValue[2] + "°C &nbsp;&nbsp; | &nbsp;&nbsp; Low " + weatherAPIKeyValue[3] + "°C</p></center></html>", SwingConstants.CENTER);
+                        JLabel pageEndLabel = new JLabel("<html><center><span style='color:white;font-size:15px;'><i>Feels like " + Integer.parseInt(weatherAPIKeyValue[1]) + "°C</i></span><p style='color:white;font-size:12px;'>High " + weatherAPIKeyValue[2] + "°C &nbsp;&nbsp; | &nbsp;&nbsp; Low " + weatherAPIKeyValue[3] + "°C</p></center></html>", SwingConstants.CENTER);
```

Comment:
I apologize for not adding details. I think it's good to have ceiling (Parsing as float and then using ceiling method) values of all the temperature numbers that we get from the API. These consists of weatherAPIKeyValue[0] [1][2]&[3].

Reply:
Alright. I did the changes. Please review.